okhttp-tests/src/test/java/okhttp3/internal/ws/WebSocketHttpTest.java

```diff
@@ -304,6 +304,9 @@
 
     clientListener.assertFailure(101, null, ProtocolException.class,
         "Expected 'Connection' header value 'Upgrade' but was 'null'");
+
+    client.connectionPool().evictAll();
```

Comment:
nit: It's better to use `TestUtil.ensureAllConnectionsReleased()`.

Reply:
Yes! Fixed.